### ISSUE TITLE
SEO Optimize

### ISSUE THREAD

**Issue by enoch85** [nextcloud/documentation#517]
Brainstorming here...

I think it would be better to only have one documentation instead of one for each version, or at least remove documentation for EOL versions. 

Right now version 9 gets the top result on Google and as a user you have to manually change URL to 12 to get the latest version. This is quite annoying + that users should always get the latest version.

Maybe add a robots rule to disallow indexing for all versions except the latest one? This would mean that the latest version would be indexed and shown on search engines, but if you would want an aold version you'd still be able to get it manually.

What do you think @jospoortvliet @MorrisJobke ?

**Comment by jospoortvliet** [comment]
Yeah, we should change this somehow. Ideally have ONE page, not version-dependent, that shows the newest, with option to show older in there.

But that means redoing our documentation generator so that ain't a minor issue... But a huge task.

**Comment by juliushaertl** [comment]
@jospoortvliet Why not just create a symlinked directory to the latest version on the web server for now. That way at least people could start linking to the newest version and hopefully those will be more prominent then in search engines.

**Comment by jospoortvliet** [comment]
@juliushaertl yeah, that works for me - not sure how doable that is. @LukasReschke probably knows - when he has time...

**Comment by MorrisJobke** [comment]
> @juliushaertl yeah, that works for me - not sure how doable that is. @LukasReschke probably knows - when he has time...

It's doable, but we need to decide what to do. We need the stable version, because they are shipped with the releases of Nextcloud. :/

**Comment by jospoortvliet** [comment]
@MorrisJobke what would need to be decided? Could we not create an alias to the latest version and forbid google from indexing any of the others?

**Comment by tflidd** [comment]
Duplicate of https://github.com/nextcloud/documentation/issues/179

**Comment by enoch85** [comment]
Right, closing this one.

**Comment by juliushaertl** [comment]
Let's use this issue since the discussion happened here.

**Comment by xraMsamohT** [comment]
Instead of cutting out the version numbers I prefer to symlink the most recent version with `latest`. So I'm able to link a mostly correct link after an upgrade to the next major version.

Old link:
https://docs.nextcloud.com/server/12/admin_manual/configuration_server/occ_command.html#versions

Current link:
https://docs.nextcloud.com/server/13/admin_manual/configuration_server/occ_command.html#versions

Permanent link:
https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#versions

---

Example:
https://help.nextcloud.com/t/mount-google-drive/2347/7

**Comment by C0rn3j** [comment]
Would also love a link to `latest`, simple because it gets tedious to keep updating links in other documentation every NC major release.

The way ghost suggested to do it (which is what Owncloud does) would be great.

https://doc.owncloud.org/server/10.0/admin_manual/maintenance/migrating.html
https://doc.owncloud.org/server/latest/admin_manual/maintenance/migrating.html

**Comment by jospoortvliet** [comment]
perhaps a think to discuss at our https://nextcloud.com/blog/join-us-at-our-next-contributor-week/ - if any of you have any skills to help set things up, you'd be totally welcome to join, discuss & help make it happen!

**Comment by skjnldsv** [comment]
The `/latest` URL has existed for some time. Robots.txt for old versions is being handled in PR #14800.
